### DISCUSSION TITLE
Judgeing Application

### DISCUSSION THREAD

**Discussion by Kevintheprogramer** [community/community#141862] [Programming Help]
### Body

who want to join my team to create a judging app

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @Kevintheprogramer, thanks for sharing with the community! 

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
